tests/e2e/utils/extension-path.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Returns the Chrome extension path.
+ * @param browser
+ * @returns path to the Chrome extension
+ */
+export const getChromeExtensionPath = async (browser: WebdriverIO.Browser) => {
+  await browser.url('chrome://extensions/');
+  const extensionItem = await $('extensions-item');
+  const extensionId = await extensionItem.getAttribute('id');
+
+  if (!extensionId) {
+    throw new Error('Extension ID not found');
+  }
+
+  return `chrome-extension://${extensionId}`;
+};
+
+/**
+ * Returns the Firefox extension path.
+ * @param browser
+ * @returns path to the Firefox extension
+ */
+export const getFirefoxExtensionPath = async (browser: WebdriverIO.Browser) => {
+  browser.url('about:debugging#/runtime/this-firefox');
```

Comment:
`await` isn't necessary here?

```suggestion
  await browser.url('about:debugging#/runtime/this-firefox');
  
```